Updated calls to RemoteConfig and RealtimeDatabase so the project now…

patm1987 opened the pull request:
… compiles

RemoteConfig moved to RemoteConfigDeprecated
instances of Firebase.Unity.Editor removed

Commit:
Updated calls to RemoteConfig and RealtimeDatabase so the project now compiles

RemoteConfig moved to RemoteConfigDeprecated
instances of Firebase.Unity.Editor removed

chkuang-g commented on Firebase_RemoteConfig/Scripts/RemoteConfigSyncBehaviour.cs:
```diff
@@ -141,7 +141,7 @@ public class RemoteConfigSyncBehaviour : MonoBehaviour {
         var targets = kv.Value;
         foreach (var target in targets) {
-          var value = FirebaseRemoteConfig.GetValue(target.FullKeyString);
+          var value = FirebaseRemoteConfigDeprecated.GetValue(target.FullKeyString);
```
`FirebaseRemoteConfig.DefaultInstance`?

chkuang-g commented on Scripts/FirebaseInitializer.cs:
```diff
@@ -74,9 +74,9 @@ public static class FirebaseInitializer {
         if (!fetching) {
 #if UNITY_EDITOR
-          var settings = FirebaseRemoteConfig.Settings;
+          var settings = FirebaseRemoteConfigDeprecated.Settings;
```
`FirebaseRemoteConfig.DefaultInstance.ConfigSettings`?
Similar to those in below

chkuang-g commented on Scripts/FirebaseInitializer.cs:
```diff
@@ -87,12 +87,12 @@ public static class FirebaseInitializer {
           Initialize(status => {
-            FirebaseRemoteConfig.SetDefaults(defaultValues);
-            FirebaseRemoteConfig.FetchAsync(TimeSpan.Zero).ContinueWith(task => {
+            FirebaseRemoteConfigDeprecated.SetDefaults(defaultValues);
```
```
Firebase.RemoteConfig.FirebaseRemoteConfig.DefaultInstance.SetDefaultAsync(defaults).ContinueWithOnMainThread( task => {
  // check if it failed...
  return Firebase.RemoteConfig.FirebaseRemoteConfig.DefaultInstance.FetchAndActivateAsync();
}).Unwrap().ContinueWithOnMainThread(task => {
      fetching = false;
      activateFetched = true;
      CallActivateFetchedCallbacks();
});
```

Commit:
Moved remote config to the new api (instance based)
In FirebaseInitializer, removed the fetching variable. It seemed to only matter if fetching was ever set to true, and it never was.
